run_pr.sh

```diff
@@ -23,6 +23,8 @@ rvm use 2.2.2
 bundle install --path ~jenkins/vendor/bundle --jobs 4 --retry 3
 bundle exec berks install
 if [ -f Thorfile ]; then
+  set -e
+  bundle exec foodcritic -f any -B ./ -G
```

Comment:
It is, but it includes -G which runs our own foodcritic gem. Some Thorfiles are missing this.

Basically, it fixes the problem of PRs passing then failing in the pipeline.